src/main/java/stsjorbsmod/memories/MemoryManager.java

```diff
@@ -196,6 +196,7 @@ public void notifyModifyMemories(Consumer<OnModifyMemoriesSubscriber> callback)
         owner.relics.forEach(possibleListener -> notifyPossibleModifyMemorySubscriber(possibleListener, callback));
         owner.powers.forEach(possibleListener -> notifyPossibleModifyMemorySubscriber(possibleListener, callback));
         this.memories.forEach(possibleListener -> notifyPossibleModifyMemorySubscriber(possibleListener, callback));
+        owner.masterDeck.group.forEach(possibleListener -> notifyPossibleModifyMemorySubscriber(possibleListener, callback));
```

Comment:
It would be good to leave a comment here noting "FreshAdventure's onSnap relies on *exactly one* version of a given uuid being notified; revisit it if we ever add hand/discard/exhaust notifications"